test/cloud-canary/mzcompose.py

```diff
@@ -116,6 +116,8 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
         # Clean up
         if args.cleanup:
             workflow_disable_region(c)
+        # Can't do restarts against cloud
+        c.down(sanity_restart_mz=False)
```

Comment:
This does not impact the cloud, what it does is shut down the local Materialize instance that was started by mzcompose.py. This instance is only used to do the version check, that is, run `mz_version()` in order to confirm that the Mz version that comes up on the cloud as a result of `enable region` is the same as `HEAD`, that is, the same version that we want to test.

Reply:
True, thanks for the pointer. New fix, verified locally.